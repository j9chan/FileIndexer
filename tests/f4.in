tests/book1.in the the% tests/book2.in