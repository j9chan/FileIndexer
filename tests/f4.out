5416 the
2512 and
2418 of
1695 to
1620 a
1233 in
877 was
861 that
788 it
728 he
